ui-participant/src/hub/documents/DocumentLibrary.tsx

```diff
@@ -13,23 +20,14 @@ import {
 import { usePortalEnv } from 'providers/PortalProvider'
 
 export default function DocumentLibrary() {
+  const { i18n } = useI18n()
   const { portal, portalEnv } = usePortalEnv()
   const { enrollees, ppUser } = useActiveUser()
-  const [participantFiles, setParticipantFiles] = useState<ParticipantFile[]>([])
-  const currentStudy = portal.portalStudies.find(pStudy =>
-    pStudy.study.studyEnvironments.find(studyEnv =>
-      studyEnv.environmentName === portalEnv.environmentName))?.study
 
-  const loadDocuments = async () => {
-    if (!currentStudy) { return }
-    const enrolleeShortcode = enrollees.find(enrollee => enrollee.profileId === ppUser?.profileId)!.shortcode
-    const documents = await Api.listParticipantFiles(currentStudy.shortcode, enrolleeShortcode)
-    setParticipantFiles(documents)
-  }
-
-  useEffect(() => {
-    loadDocuments()
-  }, [])
+  const studiesJoined = portal.portalStudies.filter(pStudy =>
+    enrollees.some(enrollee =>
+      enrollee.profileId === ppUser?.profileId && enrollee.studyEnvironmentId === pStudy.study.studyEnvironments[0].id)
```

Comment:
why do we need the profileId check here?  is that to clear out governed enrollees?

Reply:
that's the intention- i noticed we do this type of profileId in a few other areas too, and I'm wondering now if we need a more intuitive way to deal with this rather than a profileId comparison which admittedly is pretty cryptic to interpret